generators/kubernetes/templates/console/jhipster-elasticsearch.yml.ejs

```diff
@@ -186,6 +190,10 @@ spec:
       resources:
         requests:
           storage: 1Gi
+  <%_ if (kubernetesStorageClassName !== '') { _%>
```

Comment:
nit: If you decide to add .trim(), can we change this to `if (kubernetesStorageClassName)`

Reply:
done here and in all similar parts